src/test/java/org/apache/commons/dbcp2/TestDelegatingConnection.java

```diff
@@ -134,12 +134,7 @@ public void testAutoCommitCaching() throws SQLException {
     public void testCheckOpen() throws Exception {
         delegatingConnection.checkOpen();
         delegatingConnection.close();
-        try {
-            delegatingConnection.checkOpen();
-            fail("Expecting SQLException");
-        } catch (final SQLException ex) {
-            // expected
-        }
+        assertThrows(SQLException.class, () -> delegatingConnection.checkOpen(), "Expecting SQLException");
```

Comment:
Good one 👍  but this should be applied in one PR for all uses of this pattern. Rebase on git master to find the whole code base using `assertThrows()`.